Ingress/controllers/gce/Dockerfile

```diff
@@ -12,23 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# TODO: use radial/busyboxplus:curl or alping instead
-FROM ubuntu:14.04
+FROM phusion:baseimage:0.9.17
 MAINTAINER Prashanth B <beeps@google.com>
+MAINTAINER Christian Stewart <christian@paral.in>
+CMD ["/sbin/my_init"]
 
-# so apt-get doesn't complain
-ENV DEBIAN_FRONTEND=noninteractive
-RUN sed -i 's/^exit 101/exit 0/' /usr/sbin/policy-rc.d
-
-# TODO: Move to using haproxy:1.5 image instead. Honestly,
-# that image isn't much smaller and the convenience of having
-# an ubuntu container for dev purposes trumps the tiny amounts
-# of disk and bandwidth we'd save in doing so.
 RUN \
   apt-get update && \
-  apt-get install -y ca-certificates && \
-  apt-get install -y curl && \
-  rm -rf /var/lib/apt/lists/*
-
-ADD glbc glbc
-ENTRYPOINT ["/glbc"]
```

Comment:
why remove this? I mean you can, but you'll need to modify args and cmd in the yaml